Practice/Course/Assignments/ProblemsOnSorting.cs

```diff
@@ -37,6 +80,33 @@ public bool CanAttendToMeetings(List<int[]> input)
             return true;
         }
 
+        /// <summary>
+        /// Question - 3
+        /// The problems is reduced to count the overlapping intervals
+        /// Is similar with Question 2
+        /// T.C -> O(n*log(n)) - because of sorting
+        /// S.C -> O(1)
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public int CalculateConferenceRooms(List<int[]> input)
+        {
+
+            if (input.Count == 0)
+                return 0;
+
+            var countOverlap = 1;
```

Comment:
Why we start at 1? should be zero

Reply:
accommodate a meeting room for first interval